feature/platform/fabric/otg_tests/sampled_backplane_capacity_counters_test/README.md

```diff
@@ -0,0 +1,90 @@
+# gNMI-1.18: gNMI subscribe with sample mode for backplane capacity counters
+
+## Summary
+WBB is required to support gNMI Subscribe with SAMPLE or ON_CHANGE mode for various counters.
+This test if to verify that DUT supports gNMI Subscribe with sample mode, updating
+the available backplane capacity counters correctly while forwarding traffic
+
+*   Get backplace capacity before any traffic is sent
+
+*   Send some traffic and wait for the sample to be collected
+
+*   Send more traffic and wait for the sample to be collected
+
+## Testbed type
+
+*   https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed
+
+## Procedure
+
+### gNMI-1.18.1 [TODO: https://github.com/openconfig/featureprofiles/issues/2322]
+
+*   Connect DUT port-1 and 2 to ATE port-1 and 2 respectively
+
+    *   For MFF DUT ports 1 and 2 can be on different linecards
+
+*   Configure IPv4/IPv6 addresses on the  interfaces
+
+    *   Test with two sample intervals of 10 and 15 seconds
```

Comment:
see above regarding separate subtests for the different sample rates

Reply:
reworded